### PR TITLE
[SYCL] Add support for multiple missing math ops

### PR DESCRIPTION
**Pull request by MaryaSharf**
Add support for multiple missing math ops;
    - ceilf
    - copysignf
    - cospif
    - fmaxf
    - fminf

### CODE REVIEW

File: sycl/test-e2e/DeviceLib/cmath_test.cpp [hunk 1]
```diff
@@ -151,7 +158,7 @@ template <class T> void device_cmath_test_1(s::queue &deviceQueue) {
 template <class T> void device_cmath_test_2(s::queue &deviceQueue) {
   s::range<1> numOfItems{2};
   T result[2] = {-1};
-  T ref[3] = {0, 2, 1};
+  T ref[4] = {0, 2, 1};
```
Comment (MartinWehking) [inline sycl/test-e2e/DeviceLib/cmath_test.cpp:161]: Why did you change this?

**Review by bso-intel — APPROVED** [review]
